src/test/java/htsjdk/samtools/util/EdgingRecordAndOffsetTest.java

```diff
@@ -30,9 +30,8 @@
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotSame;
+import static org.testng.Assert.assertEquals;
```

Comment:
more statics, good to get rid of the accidental use of junit, but generally we don't use static imports